local-cli/templates/HelloWorld/ios/HelloWorld/Info.plist

```diff
@@ -4,6 +4,8 @@
 <dict>
 	<key>CFBundleDevelopmentRegion</key>
 	<string>en</string>
+	<key>CFBundleDisplayName</key>
+	<string>Hello App Name</string>
```

Comment:
This is the display name so I wanted to make the replacement string very obvious. I'll rename it to "Hello App Display Name"